packages/frontend/app/services/current-user.ts

```diff
@@ -77,21 +89,30 @@ export default class CurrentUserService extends Service {
   async create(name: string): Promise<void> {
     await this.hydrateCrypto();
     const { publicKey, privateKey } = await this.crypto!.generateKeys();
+    const { publicSigningKey, privateSigningKey } = await this.crypto!.generateSigningKeys();
 
     // remove existing record
     await this.store.unloadAll('user');
 
-    await this.setIdentity(name, privateKey, publicKey);
+    await this.setIdentity(name, privateKey, publicKey, privateSigningKey, publicSigningKey);
```

Comment:
Add a version of this method that doesn't require these so that when you login with just your mnemonic, it generates a new set of signing keys.

Reply:
Added